bigtable/tests/instance_admin_emulator.cc

```diff
@@ -223,6 +264,7 @@ class InstanceAdminEmulator final
 
  private:
   std::map<std::string, btadmin::Instance> instances_;
+  std::map<std::string, btadmin::Cluster> clusters_;
```

Comment:
If we are keeping a separate collection of clusters (unrelated to the instances), then we need to change `CreateInstance()` to insert the associated clusters in this collection, and `DeleteInstance()` to delete them.

The other approach is to use the list of clusters in each `btadmin::Instance` object to implement the cluster operations, for example, to create a cluster you first find the instance and then the cluster to its `clusters()` variable.

Reply:
If you decide to go with the separate `clusters_` collection then please create a bug to update `CreateInstance()` and `DeleteInstance()`, you do no need to fix that in this PR.

Reply:
@coryan ,

> to create a cluster you first find the instance and then the cluster to its clusters() variable

I do not see `clusters()` inside `btadmin::Instance`, am I missing something?

One way to keep instance and cluster associated, is to use
`std:map<std::string, std:map<std:string, btadmin::Cluster>> instance_clusters_`

but I think, it's an over do for an emulator.
Please suggest.